Sources/Nimble/Matchers/Contain.swift

```diff
@@ -9,7 +9,7 @@ private func contain<S: SequenceType, T: Equatable where S.Generator.Element ==
     return NonNilMatcherFunc { actualExpression, failureMessage in
         failureMessage.postfixMessage = "contain <\(arrayAsString(items))>"
         if let actual = try actualExpression.evaluate() {
-            return all(items) {
+            return items.all {
                 return actual.contains($0)
```

Comment:
As long as they're internal and there is only one of these implementations. I don't have a particular preference. 

Sent from my iPhone

> On Jan 21, 2016, at 2:25 PM, Brian Croom <notifications@github.com> wrote:
> 
> In Sources/Nimble/Matchers/Contain.swift:
> 
> > @@ -9,7 +9,7 @@ private func contain<S: SequenceType, T: Equatable where S.Generator.Element ==
> >      return NonNilMatcherFunc { actualExpression, failureMessage in
> >          failureMessage.postfixMessage = "contain <\(arrayAsString(items))>"
> >          if let actual = try actualExpression.evaluate() {
> > -            return all(items) {
> > +            return items.all {
> >                  return actual.contains($0)
> Ah hmm, looking at this now it seems that there's no good reason for that change. At one point I remember I was trying to chain a function call on the collection before an all and that's why I tried changing it. That usage didn't survive, obviously.
> 
> Which way do you prefer? Free function or protocol extension? Regardless, we don't need both, you're right.
> 
> —
> Reply to this email directly or view it on GitHub.
>